tests/ctst/features/azureArchive.feature

```diff
@@ -121,10 +121,13 @@ Feature: Azure Archive
     And object "obj-2" should have the same data
     And object "obj-2" should have the tag "tag2" with value "value2"
     And object "obj-2" should have the user metadata with key "x-amz-meta-123" and value "456"
+    When i wait for <restoreDays> days
```

Comment:
nit: "When" is kind-of the start a new test, so please add an empty line to make it more readable